commands/databases.go

```diff
@@ -1152,3 +1154,308 @@ func RunDatabaseSetSQLModes(c *CmdConfig) error {
+	`
+
+	databaseFirewallAddDetails :=
+		`
+Use this command to append a single rule to the existing firewall rules of a given database. This command requires the ID of a database cluster, which you can retrieve by calling:
+
+	doctl databases list
+
+This command also requires a --rule flag. Each rule passed in to the --rule flag must be of format type:value
+	- "type" is the type of resource that the firewall rule allows to access the database cluster. The possible values for type are:  "droplet", "k8s", "ip_addr", or "tag"
+	- "value" is either the ID of the specific resource, the name of a tag applied to a group of resources, or the IP address that the firewall rule allows to access the database cluster
+
+For example:
+
+	doctl databases firewalls append d1234-1c12-1234-b123-12345c4789 --rule tag:backend
+
+This would append the firewall rule "tag:backend" for database of id d1234-1c12-1234-b123-12345c4789`
+
+	databaseFirewallRemoveDetails :=
+		`
+Use this command to remove an existing, single rule from the list of firewall rules for a given database. This command requires the ID of a database cluster, which you can retrieve by calling:
+
+	doctl databases list
+
+This command also requires a --uuid flag. You must pass in the UUID of the firewall rule you'd like to remove. You can retrieve the firewall rule's UUIDs by calling:
+
+	doctl database firewalls list <db-id>
+
+For example:
+
+	doctl databases firewalls remove d1234-1c12-1234-b123-12345c4789 --uuid 12345d-1234-123d-123x-123eee456e
+
+This would remove the firewall rule of uuid 12345d-1234-123d-123x-123eee456e for database of id d1234-1c12-1234-b123-12345c4789
+			`
+
+	CmdBuilder(cmd, RunDatabaseFirewallRulesList, "list <database-id>", "Retrieve a list of firewall rules for a given database", firewallRuleDetails+databaseFirewallRuleDetails,
+		Writer, aliasOpt("ls"))
+
+	cmdDatabaseFirewallUpdate := CmdBuilder(cmd, RunDatabaseFirewallRulesUpdate, "replace <db-id> --rules type:value [--rule type:value]", "Replaces the firewall rules for a given database. The rules passed in to the --rules flag will replace the firewall rules previously assigned to the database,", databaseFirewallUpdateDetails,
+		Writer, aliasOpt("r"))
+	AddStringSliceFlag(cmdDatabaseFirewallUpdate, doctl.ArgDatabaseFirewallRule, "", []string{}, databaseFirewallRulesTxt, requiredOpt())
+
+	cmdDatabaseFirewallCreate := CmdBuilder(cmd, RunDatabaseFirewallRulesCreate, "append <db-id> --rule type:value", "Add a database firewall rule to a given database", databaseFirewallAddDetails,
+		Writer, aliasOpt("a"))
+	AddStringFlag(cmdDatabaseFirewallCreate, doctl.ArgDatabaseFirewallRule, "", "", "", requiredOpt())
+
+	cmdDatabaseFirewallRemove := CmdBuilder(cmd, RunDatabaseFirewallRulesRemove, "remove <firerule-uuid>", "Remove a firewall rule for a given database", databaseFirewallRemoveDetails,
+		Writer, aliasOpt("r"))
```

Comment:
```suggestion
		Writer, aliasOpt("rm"))
```

Looks like we have two commands both trying to use the `r` alias. How about using `rm` for `replace`?

Reply:
good catch! I have applied the changes.